ipaserver/install/ipa_acme_manage.py

```diff
@@ -52,28 +98,17 @@ def run(self):
             print("CA is not installed on this server.")
             return 1
 
-        if self.command == Command.ENABLE:
-            directive = 'enabled'
-            value = 'true'
-        elif self.command == Command.DISABLE:
-            directive = 'enabled'
-            value = 'false'
-        else:
-            raise RuntimeError('programmer error: unhandled enum case')
-
-        with DirectiveSetter(
-            paths.PKI_ACME_ENGINE_CONF,
-            separator='=',
-            quotes=False,
-        ) as ds:
-            ds.set(directive, value)
-
-        # Work around a limitation in PKI ACME service file watching
-        # where renames (what DirectiveSetter does) are not detected.
-        # It will be fixed, but keeping the workaround will do no harm.
-        pathlib.Path(paths.PKI_ACME_ENGINE_CONF).touch()
-
-        # Nothing else to do; the Dogtag ACME service monitors engine.conf
-        # for updates and reconfigures itself as required.
+        api.bootstrap(in_server=True, confdir=paths.ETC_IPA)
+        api.finalize()
+        api.Backend.ldap2.connect()
```

Comment:
Why do Dogtag plugins create a new LDAP connection instead of using the existing LDAP connection from the default API? I have seen this before in other plugins.

Reply:
I don't understand the question. This isn't a plugin, it's using code that I think expects to be run inside the server and assumes that the API is initialized and an LDAP connection exists.